cpp/include/cudf/functions.h

```diff
@@ -754,7 +754,28 @@ gdf_error gpu_comparison(gdf_column *lhs, gdf_column *rhs, gdf_column *output,gd
 
 //takes a stencil and uses it to compact a colum e.g. remove all values for which the stencil = 0
 //The lhs column is expected to have 0 null_count otherwise GDF_VALIDITY_UNSUPPORTED is returned
-gdf_error gpu_apply_stencil(gdf_column *lhs, gdf_column * stencil, gdf_column * output);
+
+/**
+ * @brief Filters data in a column using a second, boolean column to select which elements
+ * are to be used.
+ *
+ * @note This is a stable operator, i.e. the relative order of elements in the output is
+ * the same as their order in the input
+ *
+ * @note For now, this mis-handles NULLs in the following sense: The input column _must_
+ * be nullable (otherwise it will be flat-out rejected); but all NULL values are
```

Comment:
What is the definition of `nullable`?

Reply:
Also, can the `stencil` column have nulls?

Reply:
Yes, the description of the function was updated: If there are nulls in the stencil, they are treated as "false" and those values in the data don't get copied. If there are nulls in the data itself, then
 they are carried to the output if the stencil is set to "true" for that value.